build/build-config.yml

```diff
@@ -46,6 +46,8 @@ config:
       - work-dir: "reference-adapter/mgramseva-ifix-adapter"
         image-name: "ifix-reference-adapter"
         dockerfile: "build/maven/Dockerfile"
+      - work-dir: "reference-adapter/mgramseva-ifix-adapter/src/main/resources/db"
+        image-name: "mgramseva-ifix-adapter-db"
```

Comment:
Keep the image name the same as the one above. Just add the "-db" suffix.

Reply:
made the changes